faucet/config.toml

```diff
@@ -0,0 +1,10 @@
+chain_id = "dev"
+listen_address = "0.0.0.0:0"
+mnemonic = "source bonus chronic canvas draft south burst lottery vacant surface solve popular case indicate oppose farm nothing bullet exhibit title speed wink action roast"
+num_accounts = 1
+send_amount = "1000000ugnot"
+
+[cors_config]
```

Comment:
Please use flag (and/or environment variable) rather than config file

For cors, please ignore, it will be handled proxy side! Mention the requirements in a README.md

Reply:
We can switch to flags, no problem. We only need a few of them anyways.

Do you want me to update the PR with the following changes?
- disable CORS control
- switch to flags / env instead of a file

Reply:
Yes please

Reply:
@albttx 

Added the requested changes here:
[a5604d6](https://github.com/gnolang/gnochess/pull/18/commits/a5604d67268bf14728e0338e61b9d44fbcfbb6f4)